source/DeviceList/Controller.cpp

```diff
@@ -0,0 +1,42 @@
+/*
+** Project FileShare, 2022
+**
+** Author Léo Lhuile
+**
+** Started on  Sun Feb 13 22:03:32 2022 Léo Lhuile
+** Last update Mon Aug 29 20:45:51 2022 Léo Lhuile
+**
+** DeviceList/Controller.cpp : Implementation of DeviceList/Controller class
+*/
+
+#include "DeviceList/Controller.hpp"
+
+namespace FileShare::GUI::DeviceList {
+    Controller::Controller() {
+        this->view = View::create();
+        this->view->onSelectDevice.connect([this](const std::string& device) {
+            this->onSelectDevice(device);
+        });
+        this->view->onToggleDevice.connect([this]() {
+            this->onToggleDevice();
+        });
```

Comment:
```suggestion
        this->view->onSelectDevice.connect([this](const std::string& device) {
            this->HandleDeviceSelected(device);
        });
        this->view->onToggleDevice.connect([this]() {
            this->HandleDeviceToggled();
        });
```
Open to different naming scheme, the goal is to have a different name for the trigger (`onSelectDevice`) and the action (`HandleDeviceSelected`), otherwise it gets confusing.